security/external_secrets_operator/external-secrets-operator-release-notes.adoc

```diff
@@ -10,11 +10,47 @@ The {external-secrets-operator} is a cluster-wide service that provides lifecycl
 
 These release notes track the development of {external-secrets-operator-short}.
 
-:FeatureName: The {external-secrets-operator}
-include::snippets/technology-preview.adoc[leveloffset=+1]
-
 For more information, see xref:../../security/external_secrets_operator/index.adoc#external-secrets-operator-about[{external-secrets-operator-short} overview].
 
+[id="external-secrets-operator-release-notes-1-0-0_{context}"]
+== Release notes for {external-secrets-operator} 1.0.0 (General Availability)
+
+Issued: 2025-11-03
+
+The following advisories are available for the {external-secrets-operator} 0.1.0:
+
+* link:https://access.redhat.com/errata/RHBA-2025:13182[RHBA-2025:13182]
+* link:https://access.redhat.com/errata/RHBA-2025:13134[RHBA-2025:13134]
+* link:https://access.redhat.com/errata/RHBA-2025:13133[RHBA-2025:13133]
+
+Version 1.0.0 of the {external-secrets-operator} is based on the upstream external-secrets version v0.19.0. For more information, see thelink:https://github.com/external-secrets/external-secrets/releases/tag/v0.19.0[external-secrets project release notes for v0.19.0].
+
+[id="external-secrets-operator-1-0-0-bug-fixes_{context}"]
+=== Bug fixes
+
+* Before this release, many of the APIs listed in the console for the {external-secrets-operator} were missing descriptions. With this release, the API descriptions have been added. (link:https://issues.redhat.com/browse/OCPBUGS-61081[OCPBUGS-61081])
+
+
+[id="external-secrets-operator-1-0-0-features-enhancements_{context}"]
+=== New features and enhancements
+
+*Renaming and improvements on the Operator API*
+
+With this release, the Operator API `externalsecrets.operator.openshift.io`` has been renamed to `externalsecretsconfigs.operator.openshift.io`` to avoid confusions with the external-secrets provided API that has the same name, but a different purpose. The API has also been restructured and new features are added.
```

Comment:
```suggestion
With this release, the Operator API `externalsecrets.operator.openshift.io` has been renamed to `externalsecretsconfigs.operator.openshift.io` to avoid confusions with the external-secrets provided API that has the same name, but a different purpose. The API has also been restructured and new features are added.
```